src/client/components/UserPage/UserLinkTable/ToolBar/SearchInput/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { connect, useSelector } from 'react-redux'
+import {
+  InputAdornment,
+  TextField,
+  createStyles,
+  makeStyles,
+} from '@material-ui/core'
+import debounce from 'lodash/debounce'
```

Comment:
I think the convention is to `import _ from 'lodash'` then use `_.debounce()`. Might need to find other usages of this in your branch and amend them. paging @liangyuanruo for confirmation on this convention.

Reply:
I followed this convention from our legacy code. It seems like both ways are featured on [lodash website](https://lodash.com/). We might also benefit keeping cherry-picked methods, as it minimises bundle size without the need of webpack tree-shaking.

Reply:
Waiting for confirmation as to whether to change as well!

Reply:
to benefit from tree-shaking, you'll have to use one of two syntax.

```javascript
import { debounce } from 'lodash'
import debounce from 'lodash/debounce'
```

@yong-jie we can integrate this as a [rule](https://github.com/wix/eslint-plugin-lodash/blob/HEAD/docs/rules/import-scope.md) in ESLint